kodilla-spring/src/main/java/com/kodilla/spring/portfolio/BoardConfig.java

```diff
@@ -0,0 +1,34 @@
+package com.kodilla.spring.portfolio;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+import java.util.ArrayList;
+
+@Configuration
+public class BoardConfig {
+
+    @Bean
+    public Board getBoard(TaskList toDoList, TaskList inProgressList, TaskList doneList) {
+        return new Board(toDoList, inProgressList, doneList);
+    }
+
+    @Bean(name = "toDoList")
+    @Scope("prototype")
+    public TaskList getToDoList() {
+        return new TaskList(new ArrayList<>());
+    }
+
+    @Bean(name = "inProgressList")
+    @Scope("prototype")
+    public TaskList getInProgressList() {
+        return new TaskList(new ArrayList<>());
+    }
+
+    @Bean(name = "doneList")
+    @Scope("prototype")
```

Comment:
Zastanawiałaś się, czemu tutaj jest `prototype`? Jakie to może mieć znaczenie praktyczne?